src/data/custom_iconsets.ts

```diff
@@ -0,0 +1,15 @@
+export interface CustomIcons {
+  [key: string]: { path: string; viewBox?: string };
+}
+
+export interface CustomIconsetsWindow {
+  customIconsets?: Map<string, (name: string) => Promise<CustomIcons>>;
+}
+
+const customIconsetsWindow = window as CustomIconsetsWindow;
+
+if (!("customIconsets" in customIconsetsWindow)) {
+  customIconsetsWindow.customIconsets = new Map();
```

Comment:
Should we make it a map or would an object be more user friendly?

Reply:
Why would an object be more user-friendly?

Reply:
Objects are just a little more user friendly as they are more common in JS world. Also using a map as an object won't work, but won't raise either.

```js
customIconsetsWindow.customIconsets = new Map();
customIconsetsWindow.customIconsets.put('bla', () => {})
customIconsetsWindow.customIconsets['bla'] = () => {}
```

Reply:
`window.customCards` is a list of objects. It would be nice if the interfaces were the same, but I realize a list doesn't make sense here.

My first attempt yesterday at making this work with fontawesome during our discussions failed because I assumed `customInconsets['fa:'] =`would work.
Quickly fixed, but still...

Reply:
OK, let's make it an object.